src/ceph_mon.cc

```diff
@@ -140,7 +140,8 @@ int check_mon_data_empty()
   errno = 0;
   while ((de = ::readdir(dir))) {
     if (string(".") != de->d_name &&
-	string("..") != de->d_name) {
+	string("..") != de->d_name &&
+	string("kv_backend") != de->d_name) {
```

Comment:
@bassamtabbara could you explain a little bit on what is the use case for this change? "kv_backend" is created when the monitor db is created, so its existence implies that the monitor db is already created?